src/fusion.cpp

```diff
@@ -263,6 +270,22 @@ bool ConvForwardOpDescriptor::GetOpAttr(const std::string& sym, int& val) const
     {
         val = base_desc.GetGroupCount();
     }
+    else if(sym == "f_byte_stride_nk")
+    {
+        val = f_strides[0];
```

Comment:
What about bwd/wrw? Will it work correctly?

Reply:
Fusion API introduces only forward operations like:
* Convolution Forward
* Activation Forward
* Bias Forward

This is explicitly defined in Winograd constraints:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/8c172d466d25146656ae0a53f03e82ab3cb76d30/src/md_graph.cpp#L521-L524